src/app/components/error/index.html

```diff
@@ -1,13 +1,232 @@
+            </sky-docs-visual-example-text>
+            <sky-docs-visual-example-image>
+                <div>
+                  <img src="~/assets/img/guidelines/error/error-option-security.png" />
+                </div>
+            </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        
+        <h3>Image</h3>
+        
+        <sky-docs-visual-example>
+            <sky-docs-visual-example-text>
+                <p>Include the optional image when an error message is the only content on the page. Tiles have limited space and may not be important to users' immediate tasks, so exclude the image and only display the title and description.</p>
+                <h3>Action</h3>
+                <p>As with the image, include actions when an error message is the only content on the page. When it is not, the description should provide a suggestion to help correct the error, but do not use an attention-drawing action when users may be able to complete their tasks without addressing the error.</p>
+            </sky-docs-visual-example-text>
+            <sky-docs-visual-example-image>
+                <div>
+                  <img src="~/assets/img/guidelines/error/error-usage-6.png" />
+                    <div class="caption caption-success sky-text-success">Exclude images and actions when error messages only affect portions of the content on a page.</div>
+                </div>
+            </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        
+        <h3>Description</h3>
+        
+        <div class="sky-text-block">
+            <p>Use the error description to recommend a course of action to users.</p>
+        </div>
+        
+    </sky-demo-page-content>
+        
+    <sky-demo-page-content sectionHeading="Content">
+        
+        <h3>Style and tone</h3>
+        
+            <div class="sky-text-block">
+                <p>Make error messages brief and use a conversational tone. Explain why the error occurred, and when possible, offer a solution to help users address the problem.</p>
+                <p>Errors frustrate users and indicate serious problems, so error messages must be thoughtful and helpful. Avoid humor, limit the use of "please" and "sorry", and avoid negative constructions such as "you can't" and "you don't."</p>
+            </div>
+        
+    </sky-demo-page-content>
+    
+    <sky-demo-page-content sectionHeading="Layout">
+        
+        <div class="sky-text-block">
+            <p>The error component, including all of its title and description text, is always centered within its container.</p>
+        </div>
+        
+    </sky-demo-page-content>
+        
+    <sky-demo-page-content sectionHeading="Related information">
+        
+        <h3>Components</h3>
+          <ul>
+            <li><a routerLink="/components/alert">Alert</a></li>
+            <li><a routerLink="/components/Toast">Toast</a></li>
```

Comment:
The link here is busted. I'm pretty sure we just need to lowercase the "T" in "toast" at the end of the path.

Reply:
I fixed this.